lib/nexthop_group.c

```diff
@@ -820,15 +831,15 @@ static bool nexthop_group_parse_nexthop(struct nexthop *nhop,
 static bool nexthop_group_parse_nhh(struct nexthop *nhop,
 				    const struct nexthop_hold *nhh)
 {
-	return (nexthop_group_parse_nexthop(nhop, nhh->addr, nhh->intf,
-					    nhh->nhvrf_name, nhh->labels, NULL,
-					    nhh->weight, nhh->backup_str));
+	return (nexthop_group_parse_nexthop(
+		nhop, nhh->addr, nhh->intf, nhh->onlink, nhh->nhvrf_name,
```

Comment:
style: at least here in the webui, it looks like this re-indenting has moved these args more than one indent-level to the left. my editor, at least, won't maintain that: isn't our style that the continuation is just one level "left" after an empty open-paren?

Reply:
just double checked, this seems to be how clang-format likes it which is our style God.